app/Helpers/helpers.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Helpers;
+
+use App\Models\AnnouncementGlobal;
+
+class helpers {
+    /**
+     * Return the global announcement from the database
+     *
+     * @return array
+     */
+    public static function getAnnouncement() {
```

Comment:
On second thought @AlanJIsaacson , we should probably change this method name to something like `getGlobalAnnouncement`.

It will also need changing in the blade template file.